public/less/_user.less

```diff
@@ -74,4 +74,25 @@
 			}
 		}
 	}
+
+    &.notification {
+        .octicon {
+            float: left;
+            font-size: 2em;
+        }
+        .content {
+            float: left;
+            margin-left: 7px;
+        }
+
+        .octicon-issue-opened, .octicon-git-pull-request {
```

Comment:
Aren't these defined somewhere already that we can reuse? (for the activity/dashboard feed)